src/content/selector.js

```diff
@@ -93,17 +104,20 @@ window['pctSelector'] = (function(window) {
     function updateSelect(select, alias, setDefault) {
         var options = select.options;
 
-        for (var i=0; i<options.length; i++) {
-            var option = options[i];
+        var selectDefaultAlias = !isPostPreview() || isReplyPreview();
+        if (selectDefaultAlias) {
```

Comment:
```suggestion

        if (selectDefaultAlias) {
```

Reply:
On second thought, I'm just going to go ahead and merge and add this newline in -- easier than making you do it and delaying the merge.